docs/docs/location-data-from-props.md

```diff
@@ -4,20 +4,22 @@ title: Location Data from Props
 
 ## What is location data
 
-Sometimes it can be helpful to know exactly what your app's browser URL is at any given stage. Using [@reach/router](https://github.com/reach/router) for [client-side](/docs/glossary#client-side) routing in Gatsby, `location` data represents where the app is currently, where you'd like it to go, and other helpful information. The `location` object is never mutated but `reach@router` makes it helpful to determine when navigation happens.
+Sometimes it can be helpful to know exactly what your app's browser URL is at any given stage. Because Gatsby uses [@reach/router](https://github.com/reach/router) for [client-side](/docs/glossary#client-side) routing, the `location` prop is passed to any component and represents where the app is currently, where you'd like it to go, and other helpful information. The `location` object is never mutated but `reach@router` makes it helpful to determine when navigation happens. Here is a sample `props.location`:
 
 ```js
 {
   key: 'ac3df4', // does not populate with a HashHistory!
   pathname: '/somepage',
-  search: '?someurlquery=searching-string',
+  search: '?someurlparam=foo&anotherurlparam=bar',
```

Comment:
```suggestion
  search: '?someurlparam=valuestring1&anotherurlparam=valuestring2',
```

In the interest of clarification, I think it would be nice to spell these out.

Reply:
`foo` and `bar` are standard [metasyntactic variables](https://en.wikipedia.org/wiki/Metasyntactic_variable) and unlike `valuestring1` and `valuestring2`, don't suggest that only 2 parameter might be allowed.